task_DEV_6/task_DEV_6/CustomStack.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+
+namespace task_DEV_6
+{
+    class CustomStack
+    {
+        private int[] arrayStack;
+        private int count;
+        private const int ARRAYLENGTH = 10;
+
+        public CustomStack()
+        {
+            arrayStack = new int[ARRAYLENGTH];
+        }
+
+        public CustomStack(int item)
+        {
+            arrayStack = new int[item];
+        }
+
+        public int Capacity()
+        {
+            return arrayStack.Length;
+        }
+
+        public int Count()
+        {
+            return count;
+        }
+
+        public bool IsEmpty()
+        {
+            if (count == 0)
+            {
+                return true;
+            }
+            return false;
+        }
+
+        public void Resize()
```

Comment:
Why `public`?

Reply:
Method `Resize` shouldn't contain logic for count check.

Reply:
Done.